tests/test_init_and_backend.py

```diff
@@ -0,0 +1,69 @@
+import os
+from os import error
+import unittest
+from pathlib import Path
+import sys
+import tempfile
+import shutil
+import filecmp
+
+sys.path.insert(0, os.getcwd() +'/../')
```

Comment:
Are the path shenanigans strictly necessary? When you run the test module from the top-level of the repo `mkosi` should be in the module search path.

Manually changing the search path tends to be brittle.